spec.html

```diff
@@ -34690,7 +34690,7 @@ <h1>Properties of _TypedArray_ Instances</h1>
 
 <!-- es6num="23" -->
 <emu-clause id="sec-keyed-collection">
```

Comment:
should this clause ID also now be plural? (also we'd want to retain the old link)

Reply:
It didn't seem to me like the clause id needed to be pluralized, but it's up to the editors. (We'd retain the old id by moving it to an 'oldids' attribute.)

Reply:
ah ok awesome. could we do that? (Pluralize the id and add the oldid)

Reply:
Done!

Reply:
I'd prefer not introducing oldids unless the clause title changes significantly.

Reply:
(For future reference, this one is fine :))